elf-size-analyze.py

```diff
@@ -1039,6 +1069,23 @@ def print_tree(header, symbols):
         for line in lines:
             line.print()
```

Comment:
`print_tree` and `create_json` share most of their code. All the tree-creation logic can be extracted into a separate function, e.g. `def prepare_tree(symbols)`, which would return `tree`. Then the original functions would take `tree` as parameter, so `def print_tree(tree, header)` and `def create_json(tree)`.